internal/cosign/verify.go

```diff
@@ -0,0 +1,327 @@
+//
+// Copyright 2023 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//nolint:goheader // code from Sigstore
```

Comment:
just to understand why did you copy the code?

Reply:
Hi Carlos! I noticed that in other projects this module is imported directly (example: https://github.com/kubernetes-sigs/release-sdk/blob/4ff121c553b87c042aa57aaf7f87db05831791bd/sign/impl.go#L32) . However, that function literally prints on stdout the signature verification result. I did not want this behavior and so I copied the file and removed the printing part.

Reply:
thanks!